tests/getsmarter_api_clients/test_geag.py

```diff
@@ -53,4 +59,66 @@ def test_get_terms_and_policies(self):
         response = client.get_terms_and_policies()
         self.assertEqual(response, terms_and_conditions)
         self.assertEqual(len(responses.calls), 1)
-        self.assertEqual(responses.calls[0].request.url, f'{self.api_url}/terms')
+        self.assertEqual(responses.calls[0].request.url, self.terms_url)
+
+    @responses.activate
+    def test_create_allocation(self):
+        responses.add(
+            responses.POST,
+            self.allocations_url,
+            status=204,
+        )
+        client = GetSmarterEnterpriseApiClient(**self.mock_constructor_args)
+
+        kwargs = {
+            'payment_reference': 'payment_reference',
+            'address_line1': '10 Lovely Street',
+            'city': 'Herndon',
+            'postal_code': '35005',
+            'state': 'Alabama',
+            'state_code': 'state_code',
+            'country': 'country',
+            'country_code': 'country_code',
+            'first_name': 'John',
+            'last_name': 'Smith',
+            'email': 'johnsmith@example.com',
+            'date_of_birth': '2000-01-01',
+            'mobile_phone': '+12015551234',
+            'work_experience': 'None',
+            'terms_accepted_at': '2022-07-25T10:29:56Z',
+            'currency': 'USD',
+            'order_items': [
+                {
+                    'productId': '87c24e19-b82c-4acd-ab90-714af629f11a',
```

Comment:
nit: Is it worth adding a comment on this line to explicitly denote that this UUID is actually a `variant` UUID for future us?